aws/resource_aws_nat_gateway_test.go

```diff
@@ -161,3 +189,141 @@ resource "aws_route_table_association" "public" {
     route_table_id = "${aws_route_table.public.id}"
 }
 `
+
+const testAccNatGatewayConfigTags = `
+resource "aws_vpc" "vpc" {
+    cidr_block = "10.0.0.0/16"
```

Comment:
Small nitpick: can you fix the indentation through the tests here? perhaps using 2 spaces, so that it is consistent with other ones?

Thanks!

Reply:
Done.